.github/workflows/ci.yml

```diff
@@ -43,6 +43,32 @@ jobs:
         run: bundle exec danger
         env:
           GITHUB_TOKEN: ${{ secrets.DANGER_GITHUB_API_TOKEN }}
+  
+  run-swiftformat:
+    name: Run SwiftFormat Linting
+    runs-on: macos-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set build image var
+      run: echo "ImageVersion=$ImageVersion"  >> $GITHUB_ENV
+    - name: Cache RubyGems
+      uses: actions/cache@v2
+      id: rubygem-cache
+      with:
+        path: vendor/bundle
+        key: ${{ runner.os }}-${{ env.ImageVersion }}-gem-${{ hashFiles('**/Gemfile.lock') }}
+        restore-keys: ${{ runner.os }}-${{ env.ImageVersion }}-gem-
+    - name: Cache Mint
+      uses: actions/cache@v2
+      id: mint-cache
+      with:
+        path: /usr/local/lib/mint
+        key: ${{ runner.os }}-mint-${{ hashFiles('./Mintfile') }}
+        restore-keys: ${{ runner.os }}-mint-
+    - name: Run bootstrap.sh
+      run: ./bootstrap.sh
+    - name: Run SwiftFormat
+      run: ./Scripts/swiftformat.sh
```

Comment:
Hmmm, I don't think we should create a new worker for this. It literally spends minutes initializing and 2 seconds running the actual script :) Can we move this to the DANGER action instead?

Reply:
Yes I can move to the danger job, but maybe that is confusing and people think this is part of danger ? Because it is not, if linting fails, danger won't post a comment anywhere. Also this linting is not run by danger so the job name is would be a bit misleading 🤔 What if we renamed the job to "Automated PR Review", or something like that

Reply:
@VojtaStavik Let me know if this is okay now: https://github.com/GetStream/stream-chat-swift/pull/871/commits/199b50f4cc2d08e39b7760ba4be8bbc69944b328

I could not setup swift format in danger, because there's not much information about that plugin. It looks it expects to set the binary path of swiftformat in the config file, and they recommend putting the binary in our repo, but we use Mint to manage swiftformat. So it is not clear on danger plugin' repo on how we can set it up with Mint.  

Maybe we can create a Ticket to investigate this further?

Reply:
Sounds good regarding the ticket 👍 Regarding the linter script location - I see your point. It's up to you. I wouldn't worry about it too much since we will solve it in CIS-684 🤷 So yeah, either keep it as it, or move it and rename.